### DISCUSSION TITLE
Error compile LISF LDT - fortranc.mod

### DISCUSSION THREAD

**Discussion by laurizioA** [NASA-LIS/LISF#1200] [General]
Compiling LIS 7.3.5

I trying to install LISF in Ubuntu 20.04.5 LTS.

When I try to compile the following error occurs.

gfortran -c -pass-exit-codes -ffree-line-length-0 -O2   -fconvert=big-endian -DGFORTRAN -I/home/codesal/anaconda3/envs/lis/mod  -I/home/codesal/anaconda3/envs/lis/include/ -I//home/codesal/Build_WRF/LIBRARIES/netcdf/include/  -I/home/codesal/anaconda3/envs/lis/include/  -I/usr/local/hdf4/include/  -I/home/codesal/anaconda3/envs/lis/include/  -I/home/codesal/anaconda3/envs/lisf/include/ -I/home/codesal/anaconda3/envs/lisf/include/ -I./ -I../lib/xmrg/ -I../params/VIC/ -I../USAFSI/ ../params/soils/read_ISRIC_soilfractions.F90
../params/soils/read_ISRIC_soilfractions.F90:117:8:

  117 |     use fortranc
      |        1
Fatal Error: Cannot open module file ‘fortranc.mod’ for reading at (1): Arquivo ou diretório inexistente
compilation terminated.
make: *** [Makefile:108: read_ISRIC_soilfractions.o] Erro 1

[ERR] Compile failed
-------------------------------------------------------------------

I searched about fortranc but didn't find anything so far.

Thank you all

**Comment by MariRiba** [comment]
Hi 
I am also getting the same error in LISF V7.5.0 . Did you find the solution.?

**Reply by MariRiba** [reply]
Reinstalling FortranGIS solved the problem/error.
:)

**Reply by jvgeiger** [reply]
Hello.  @MariRiba is correct.  Reinstall FortranGIS.

Or, when running the configure script for LDT, you may answer

`Enable GeoTIFF support? (1-yes, 0-no, default=1): 0`

But this will disable support for reading ISRIC soil fraction and ISRIC texture data.
